domain-models-runtime/src/main/java/org/folio/rest/impl/TenantAPI.java

```diff
@@ -283,10 +283,12 @@ private void completeJob(TenantJob job, Context context) {
           }
           return loadData(tenantAttributes, job.getTenant(), headers, context);
         })
-        .onComplete(res -> {
-          if (res.failed()) {
-            job.setError(res.cause().getMessage());
+        .onFailure(cause -> {
+          String message = cause.getMessage();
+          if (message == null) {
+            message = cause.getClass().getName();
           }
+          job.setError(message);
```

Comment:
```suggestion
          job.setError(cause.getClass().getName() + " : " + cause.getMessage());
```

Reply:
The class name is not very useful in most cases, so I prefer Julian's approach (not to mention it preserves behavior to what it it is now -- some unit tests could break of this).

Reply:
This fixes the problem, but always adds unnecessary information: If getMessage() is nul it adds "null", otherwise the message gives good information and the exception class name doesn't add any relevant information.